addon/templates/components/session-overview.hbs

```diff
@@ -242,8 +242,53 @@
         </span>
       </div>
     {{/if}}
+    <div class="postrequisite block">
+      <label>
+        {{#if session.hasPostrequisite}}
+          {{#link-to "session.index" (await session.postrequisite.course) (await session.postrequisite)}}
+            {{fa-icon  "external-link-square-alt"}} {{t "general.duePriorTo"}}:
+          {{/link-to}}
+        {{else}}
+          {{t "general.duePriorTo"}}:
+        {{/if}}
+      </label>
+      {{#if editable}}
+        {{#editable-field
+          value=(await postRequisiteTitle)
+          save=(action "changePostrequisite")
+          close=(action "revertPostrequisiteChanges")
+        }}
+          <select onchange={{action "setPostrequisite" value="target.value"}}>
+            <option value="null" selected={{is-empty (await session.postrequisite)}}>{{t "general.none"}}</option>
+            {{#each (await linkablePostrequisites) as |linkablePostrequisite|}}
+              <option value={{linkablePostrequisite.id}} selected={{is-equal linkablePostrequisite (await session.postrequisite)}}>
+                {{linkablePostrequisite.title}}
+              </option>
+            {{/each}}
+          </select>
+        {{/editable-field}}
+      {{else}}
+        {{#if session.hasPostrequisite}}
+          {{session.postrequisite.title}}
+        {{else}}
+          {{t "general.none"}}
+        {{/if}}
+      {{/if}}
+    </div>
+    <div class="prerequisites block">
+      <label>{{t "general.prerequisites"}}:</label>
+      {{#if session.hasPrerequisites}}
+        <span>
+          {{#each (await session.prerequisites) as |prerequisite|~}}
```

Comment:
I don't know what the `~` does here - extra empty char?

Reply:
the tilde is for whitespace control, this got ported over from handlebars. check it out!
https://handlebarsjs.com/expressions.html#whitespace-control